jobs/mongodb_migration/src/mongodb_migration/collector.py

```diff
@@ -276,4 +278,17 @@ def get_migrations(self) -> list[Migration]:
                 version="20231106193200",
                 description="add 'partial', 'num_rows' and 'num_bytes' fields for 'split-duckdb-index' cache records",
             ),
+            MigrationDeleteJobsByStatus(
+                status_list=["success", "error", "cancelled"],
+                version="20231201074900",
+                description="delete jobs with success, error and cancelled status",
+            ),
+            MigrationQueueDeleteTTLIndex(
```

Comment:
remark: I think that we will have to deploy twice to have this index deleted (since during the first sync, old pods will still be running and will recreate the deleted index)

Reply:
we could instead:
1. create another PR that only removes the index from the code in queue.py, and deploy
2. in this PR: delete the index + the rest of unrelated changes